tests/test_pip.py

```diff
@@ -106,68 +106,58 @@ def test_no_duplicate_constraints_pex_warnings(
     )
 
 
+@pytest.mark.skipif(
+    not IS_LINUX
+    or not any(
+        (
+            "manylinux2014_x86_64" == platform.platform
+            for platform in PythonInterpreter.get().supported_platforms
+        )
+    ),
+    reason="Test requires a manylinux2014_x86_64 compatible interpreter.",
+)
 @applicable_pip_versions
 def test_download_platform_issues_1355(
```

Comment:
This test changed substantially since ansicolors is now an example of a distribution which is "cross-buildable" for foreign platforms, but it did so to reflect the actual original issue: https://github.com/pantsbuild/pex/issues/1355

That issue is now verified directly and exactly.